client/src/components/SaveVideo.js

```diff
@@ -4,14 +4,15 @@ import axios from 'axios';
 
 class SaveVideo extends Component {
 	state = {
```

Comment:
remove SaveVideo component since saving video now resides in the video detail component